src/legacy/core_plugins/kibana/public/dashboard/np_ready/types.ts

```diff
@@ -93,11 +92,11 @@ export type SavedDashboardPanelTo60 = Pick<
   readonly type: string;
 };
 
-export type DashboardAppStateDefaults = DashboardAppStateParameters & {
+export type DashboardAppStateDefaults = DashboardAppState & {
```

Comment:
// nit 

`DashboardAppState` is used before its definition

Reply:
fixed